onnxruntime/core/common/cpuid_info.h

```diff
@@ -17,7 +17,7 @@ namespace onnxruntime {
 
 class CPUIDInfo {
  public:
-  static const CPUIDInfo& GetCPUIDInfo() {
+  static CPUIDInfo& GetCPUIDInfo() {
```

Comment:
We shouldn't jeopardize const safety in code that runs in production to satisfy our testing requirements. We'll have to think of another way to mock this.

Reply:
The removing of const is an expected behavior to remind callers that there is now a mocking function that could CHANGE the returned instance. For sure we could restore the const by making the is_hybrid_ a mutable, but it is misleading since indeed the instance could be altered anytime calling of MockHybrid(...)